sw/device/silicon_creator/rom_ext/sigverify_keys.c

```diff
@@ -0,0 +1,186 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
If I'm reading this correctly, `sigverify_keys.[ch]` enforce key policy.  Should this perhaps be a library in `silicon_creator/lib`?

Reply:
yes, there are some things that can be moved into a library. Should I do that in this PR, or is it ok to make that change in a follow up change?

Reply:
I'll leave that decision up to you.  Its ok with me to make that change in a follow-up PR.

Reply:
SG. I will leave it for a follow up PR since there is still some work needed to decide how to define a key invalidation flow for ROM_EXT.